File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.js
function solution(s) {
    let arr = s.split(" ");
    let i = 0;
    for (i=0; i<arr.length; i++){
       arr[i] = Number(arr[i]); 
    }
    
    let answer = "";
    answer += Math.min(...arr).toString()
    answer += " "
    answer += Math.max(...arr).toString()
    
    return answer
}

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.cpp
#include <iostream>
#include <string>
#include <vector>
#include <sstream>
#include <algorithm>

using namespace std;

string solution(string s) {
    
    stringstream ss(s);
    
    vector<int> words;
    string word;
    
    while(getline(ss, word, ' ')) {
        words.push_back(stoi(word));
    }
    
    sort(words.begin(), words.end());
    string answer;
    answer += to_string(words[0]);
    answer += " ";
    answer += to_string(words[words.size()-1]);
    
    return answer;
}